README.md

```diff
@@ -311,6 +316,11 @@ You can specify dateShift as a de-identification method in the configuration fil
 2. If the input date/dateTime/instant value is indicative of age over 89, it will be redacted (including year) according to HIPAA Safe Harbor Method.
 3. If the input dateTime/instant value contains time, time will be redacted. Time zone will keep unchanged.
 
+## Crypto-hash method
+You can specify the crypto-hash method in the configuration file. We use HMAC-SHA256 algorithm, which outputs a Hex encoded representation of the hashed output (for example, ```a3c024f01cccb3b63457d848b0d2f89c1f744a3d```). Considering anonymization output conformance, we recommend you not to specify crypto hash method for FHIR elements with special format requirements like *date*, *datetime*, *url*, etc.
+
+A typical scenario is to replace resource ids across FHIR resources via crypto hashing. With a specific hash key, same resource ids that reside in resources and references will be hashed to a same value. There is a special case when crypto hashing a [literal reference](https://www.hl7.org/fhir/references.html#literal) element. We capture and transform only the id part from a reference, for example, reference ```Patient/123``` will be hashed to ```Patient/a3c024f01cccb3b63457d848b0d2f89c1f744a3d```. In this way, you can easily resolve references across anonymized FHIR resources.
+

```

Comment:
The tool captures and transforms only the id part ......

Reply:
Fixed, thanks.